test/units/testsuite-13.machinectl.sh

```diff
@@ -23,9 +23,9 @@ trap at_exit EXIT
 systemctl service-log-level systemd-machined debug
 systemctl service-log-level systemd-importd debug
 
-# Mount tmpfs over /var/lib/machines to not pollute the image
+# Mount temporary directory over /var/lib/machines to not pollute the image
 mkdir -p /var/lib/machines
-mount -t tmpfs tmpfs /var/lib/machines
+mount --bind "$(TMPDIR=/var/tmp mktemp -d)" /var/lib/machines
```

Comment:
```suggestion
mount --bind "$(mktemp --tmpdir=/var/tmp -d)" /var/lib/machines
```

might be a slightly better choice.